Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2015-06-12 -- v3.0.2
+- Add support for global data collection switch in FIRApp.
+- Adding warning message when topic format is not right.
+- Adding deprecation warning to suppress deprecated token method usage inside FIRMessaging.
```

Comment:
Consider rephrasing to `Silenced a deprecation warning in FIRMessaging.`

Reply:
Done.